docs/app-center-preparing-assets.md

```diff
@@ -0,0 +1,124 @@
+---
+id: app-center-preparing-assets
+title: App Center
+sidebar_label: Preparing assets
+---
+
+
+##### Preparing assets for publishing your Aragon app
+
+Before you publish your Aragon app in the App Center, you need to prepare a few types of assets to display it correctly, and differentiate from other apps.
+
+ The main assets are: 
+- App Menu Icon (SVG)
+- Large app icon (SVG or PNG)
+- App Badge Icon (SVG or PNG)
+- Screenshots (PNG)
+- Short description (text only)
+- Long description (Markdown or text)
+
+<br>
+In the next sections we will go through these one by one, as well as link to some templates so it’s easy for you to prepare everything that you need.
+
+<br>
+
+
+## Icons
+
+### Menu icon
+
+This icon is used on the left hand app navigation panel of Aragon.
+<br>
+<br>
+<span>![*](/docs/assets/check.svg) **Size** of the icon including padding is **22x22 px**. The padding should be around 3px or more on each side, depending on the icon shape</span>
+
+<span>![*](/docs/assets/check.svg) **Color** of the icon should be **#888888** so that we can automatically colorize the icon for you correctly in the Aragon client for various icon states</span>
+
+ <span>![*](/docs/assets/check.svg) **Line width** of the icon shape should be 1px so your app icon will look consistent with others. Sticking to the pixel grid is also recommended for sharp rendering</span>
+
+<span>![*](/docs/assets/check.svg) Export to **SVG** for App Center submissions</span>
+
+<br>
+
+<center> ![Do](/docs/assets/app-center-1.svg) ![Dont](/docs/assets/app-center-2.svg) </center>
+
+### Large app icon
+
+The large app icon will be used on the Aragon App Center, and unique icons help users to visually differentiate between apps. The large app icons can be displayed in various sizes, so you should target at least 192x192px when designing the icon.
+<br>
+<br>
+<span>![*](/docs/assets/check.svg) Try to have some visual similarity to the App menu icon - otherwise users can be confused    </span>
```

Comment:
```suggestion
<span>![*](/docs/assets/check.svg) Try to have some visual similarity to the App menu icon - otherwise users may be confused after installing the app    </span>
```